lib/Report/SummaryReport.php

```diff
@@ -922,6 +922,49 @@ private function getSummaryReportMongoDb(
             'query' => $cursorPeriodQuery
         ]);
 
+        if (count($displayIds) > 0) {
```

Comment:
We have to duplicate this whole block to get the displayIds in?

Can't we just append to the array?

```
$matchExpr[] = [
    '$in' => [ '$displayId', $displayIds ]
];
```